### PR TITLE
feat(multiarch): adding multi arch build process for LVM Driver

### PR DESCRIPTION
**Pull request by pawanpraka1**
Signed-off-by: Pawan <pawan@mayadata.io>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: v0.4.2
```
Comment (akhilerm) [inline .github/workflows/build.yml:78]:
```suggestion
          version: v0.5.1
```
`0.5.1` can be used.

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: v0.4.2
```
Comment (akhilerm) [inline .github/workflows/pull_request.yml:70]:
```suggestion
          version: v0.5.1
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: v0.4.2
```
Comment (akhilerm) [inline .github/workflows/release.yml:47]:
```suggestion
          version: v0.5.1
```

File: .travis.yml [hunk 4]
```diff
@@ -20,6 +20,11 @@ addons:
 before_install:
   - sudo apt-get update -qq
 install:
+  - if [ "$TRAVIS_BUILD_DIR" != "$GOPATH/src/github.com/openebs/lvm-localpv" ]; then
```
Comment (akhilerm) [inline .travis.yml:23]: Why do we need the travis workflow. @pawanpraka1 ?

Comment (pawanpraka1) [reply .travis.yml:23]: To run the test cases on the forked repo.

Comment (akhilerm) [reply .travis.yml:23]: Can we remove them and use github actions on the fork also. IMHO, keeping it just for running tests on fork doesn't sound like a great way!

Comment (pawanpraka1) [reply .travis.yml:23]: can we setup minikube via github action?

Comment (akhilerm) [reply .travis.yml:23]: Last time I tried, minikube was not working. If we need minikube for testing then we should keep travis. I thought it was only for unit tests.

File: .travis.yml [hunk 5]
```diff
@@ -20,6 +20,11 @@ addons:
...
+    mkdir -p $GOPATH/src/github.com/openebs/;
+    mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/openebs;
+    cd $GOPATH/src/github.com/openebs/lvm-localpv;
+    fi
   - make bootstrap
   - make format
   - make license-check
```
Comment (akhilerm) [inline .travis.yml:30]: Can the license check also be included in the github actions pullrequest workflow.

**Review by akhilerm — APPROVED** [review]
changes looks good to me.

**Review by akhilerm — APPROVED** [review]

**Review by shubham14bajpai — APPROVED** [review]
lgtm

**Review by kmova — APPROVED** [review]

**Pull request merged**